Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -1134,7 +1134,7 @@ MonoBehaviour:
         id: 0
         description: None
         axisConstraint: 0
-      keyCode: 346
+      keyCode: 338
```

Comment:
Why the button change in the asset when the mappings have not changed?

Reply:
This was left over when you reverted a change to the mappings during the review of #5265 but didn't update the profiles as well.

Reply:
Just doubled checked by creating a new asset and you are indeed correct, 338/339 are the correct values. Great catch.